README.md

```diff
@@ -169,3 +169,7 @@ for unit tests only
 ```bash
 make test-unit
 ```
+
+### Releasing
```

Comment:
Could you please explain by words what the release script does?

Reply:
Sure.